src/librbml/lib.rs

```diff
@@ -167,6 +275,15 @@ pub mod reader {
         pub next: uint
     }
 
+    pub fn tag_at(data: &[u8], start: uint) -> DecodeResult<Res> {
+        let v = data[start] as uint;
+        if v < 0xf1 { // 0xf0 is intentionally left out
```

Comment:
What does this comment mean? (It seems that `v == 0xf0` is included in this case, so describing it as left-out is confusing to me.)